packages/manager/modules/pci/src/projects/project/project.service.js

```diff
@@ -25,4 +25,33 @@ export default class {
       .get(`/cloud/project/${serviceName}/region`)
       .then(({ data: regions }) => regions);
   }
+
+  getStorageRegions(projectId, regionCapacity) {
+    return this.getRegions(projectId).then((regions) => {
+      return regions.filter(({ services }) =>
+        services.find(({ name }) => name === regionCapacity),
+      );
+    });
+  }
+
+  getS3StorageRegions(projectId, regionCapacity) {
+    return this.getRegions(projectId).then((regions) => {
+      return regions.filter(({ services }) =>
+        services.find(
+          ({ name }) =>
+            name === regionCapacity[0] || name === regionCapacity[1],
+        ),
+      );
+    });
+  }
+
+  getRegions(projectId) {
+    return this.$http
+      .get(`/cloud/project/${projectId}/region`, {
+        headers: {
+          'X-Pagination-Mode': 'CachedObjectList-Pages',
+        },
+      })
+      .then(({ data }) => data);
+  }
```

Comment:
There is already a function which makes a call to the same API. It can be re-used. `getCustomerRegions`

Reply:
Hi Anoop, there is a change in request headers. If I use getCustomerRegions its only gives region names. I need complete details of each regions.

Reply:
Hence not able to use getCustomerRegions call

Reply:
done Anoop